Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,55 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            double result=0.0;
+            int index=0;
+            string tempString="";
+            string[] subEq=Regex.Split(equation,"=");
+            subEq[0]=subEq[0].Replace('*','=');
+            string[] substr=Regex.Split(subEq[0],"=");
+            
+            if (subEq[0].Contains("?"))

```

Comment:
Use single responsibility principle and break the functionality in various functions.